.ci/docker/ci_commit_pins/vllm.txt

```diff
@@ -0,0 +1 @@
+29d1ffc5b4c763ef76aff9e3f617fa60dd292418
```

Comment:
I think this should be added to `.github/ci_commit_pins` instead because this is built after PyTorch, so we couldn't build this as part of Docker build workflow.  But then, I think it depends on how you want to use this file later on as I remember you mention that we want to do the Docker image route eventually